src/components/ExperienceTable/ExperienceTable.tsx

```diff
@@ -1,80 +1,115 @@
-        <thead>
-          <tr className="border-b border-inactive-gray text-left text-body-md">
-            <th scope="col" className="font-normal pb-3">
-              {mainColumnHeader}
-            </th>
-            {sm && (
-              <>
-                <th scope="col" className="font-normal">
-                  {t('started')}
-                </th>
-                <th scope="col" className="font-normal">
-                  {t('ended')}
-                </th>
-                <th scope="col" className="font-normal">
-                  {t('competences')}
-                </th>
-              </>
-            )}
-          </tr>
-        </thead>
-        <tbody>
-          <SpacerRow />
-          {categorizedRows.map((row) => (
-            <React.Fragment key={row.key}>
-              <ExperienceTableRow key={row.key} row={row} onRowClick={onRowClick} className="bg-white" />
-              {row.subrows?.map((subrow, i) => (
+    <div className="overflow-x-auto">
+      {rows.length > 0 && (
+        <table className="w-full" border={0} cellPadding={0} cellSpacing={0}>
+          <thead>
+            <tr className="border-b border-inactive-gray text-left text-body-md">
+              <th scope="col" className="font-normal pr-7 pb-3">
+                {mainColumnHeader}
+              </th>
+              {sm && (
+                <>
+                  <th scope="col" className="font-normal pr-7 pb-3">
+                    {t('started')}
+                  </th>
+                  <th scope="col" className="font-normal pr-7 pb-3">
+                    {t('ended')}
+                  </th>
+                  <th scope="col" className={`font-normal pb-3 ${onNestedRowClick ? 'pr-7' : 'pr-5'}`.trim()}>
+                    {t('competences')}
+                  </th>
+                </>
+              )}
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td colSpan={5} className="py-3"></td>
+            </tr>
+            {categorizedRows.map((row, index) => (
+              <React.Fragment key={row.key}>
                 <ExperienceTableRow
-                  key={subrow.key}
-                  row={subrow}
+                  key={row.key}
+                  row={row}
                   onRowClick={onRowClick}
-                  className={i % 2 !== 0 ? 'bg-white' : 'bg-bg-gray'}
-                  nested
+                  className="bg-white border-spacing-x-2"
                 />
-              ))}
-              <SpacerRow className="py-5" />
-            </React.Fragment>
-          ))}
+                {row.subrows?.map((subrow, i) => (
+                  <ExperienceTableRow
+                    key={subrow.key}
+                    row={subrow}
+                    last={i === row.subrows!.length - 1}
+                    onRowClick={onNestedRowClick}
+                    className={i % 2 !== 0 ? 'bg-white' : 'bg-bg-gray'}
```

Comment:
It seems that the design uses `white` color as a background color but with the **opacity** value of `0.6`.

When looking with color inspector, close color hexcode is `#f9fafb`